packages/mail/lib/getNewsletterSettings.js

```diff
@@ -37,10 +38,7 @@ module.exports = async ({ user }, NewsletterSubscription) => {
   const subscriptions = []
   supportedInterestConfigs.forEach(({ interestId, visibleToRoles }) => {
     // only return visible interests
-    const subscribed =
-      status === MailchimpInterface.MemberStatus.Subscribed
-        ? !!member.interests[interestId]
-        : false
+    const subscribed = !!member.interests[interestId]
     if (
       subscribed ||
```

Comment:
To build newsletter subscription even if `subscribed` is false, I believe this check has to go.
```suggestion
```